gf-guide/onboarder-workflow.md

```diff
@@ -39,289 +39,794 @@ This guide will help team members to understand the validation process of a font
+<figcaption aria-hidden="true">From the issue to fonts.google.com</figcaption>
 </figure>
 
-The issues should follow these rules:
+This graphic shows how a project travels from the google/fonts github repository to [fonts.google.com](https://fonts.google.com).
 
--   **One issue per font family.**
+- The [google/fonts repository](https://github.com/google/fonts) is linked to the `dev-sandbox` server. Each PR that are merged will appear few minutes or more after on the `dev-sandbox` (if it works correctly).
+- Once a PR is merged, the linked issue gets automatically closed, but it doesn't mean that the issue is completed, because the fonts are not live yet.
+- Once the fonts are in the `dev-sandbox`, we can add them to the [to_sandbox.txt list](https://github.com/google/fonts/blob/main/to_sandbox.txt). The API engineers then push the list of families in `sandbox`. 
+- From there an onboarder can check if the fonts look alright in `sandbox`, and if it is the case, they can add them to the [to_production.txt list](https://github.com/google/fonts/blob/main/to_production.txt). The API engineers can then push the families onto the [fonts.google.com](https://fonts.google.com), availabe to the public.
 
-    Onboarders can also use the issues as a progress tracker of their assigned projects by adding a checklist in the first comment and regular notes.
+This process takes 2 to 4 weeks, because the push to sandbox or production only happens in alternance once a week — if not less.
 
--   **They have labels**.
+You see in the graphic the importance of the issues and the pull requests (PR). We try to keep connected the issue and PR that fixes it but we do track them in different boards. The **Google Fonts board tracks the issues** (that are our link to the upstream repos), the **Traffic Jam board tracks the PRs** (that are our link to the servers and the API).
 
-    At least a *primary category label* (start with `I`). Further information about labels below.
-    
-    E.g <mark class=grey>I Font bug</mark>.
+Finally, it is important to note that the **API is the primary space for the users**, the **google/fonts repo is primary space of the onboarders**, the **upstream font repo is the primary space of the author**. Since it is impossible to satisfy everyone in all spaces, these priorities are to take seriously into consideration when facing a problem: 
+- All fonts sent to the users should be tailored for the best user experience.
+- GF shouldn't force anything onto the designer's repository. A consensual collaboration is key. In case of disagreement, GF would need permission to migrate the upstream repo into the `googlefonts` org.
+- The google/fonts issue tracker can harvest users' feedback and reports, but this repository is primarily the workspace of an entire team. A cordial and respectful communication is necessary, but the user is not king there: the mental health of the team members is.
```

Comment:
This might be too pessimistic and negative. Being a team member is a HUGE privledge, If you don't like working with the public then don't work on open-source software, there are pleanty of people that LOVE the idea of open-source and want to work with the public.

Reply:
I really don't see the link between acknowledging that this is the workspace of actual people and answering issues. I have been answering almost all issues in the past years. But it doesn't mean that you have to submit to all users requests as some would expect, and it doesn't mean this repo exists for the users: it is a workspace of the engineers and the onboarders, and a complicated database. You can answer issues and acknowledge at the same time that this space is where some people spend 8h a day everyday to make things work, and their mental health need to be preserved and respected in order for things to continue working. 

Also being a team member is not a privilege, it is a job for which you get paid.